package.json

```diff
@@ -104,7 +104,7 @@
     "postbuild": "npm run sitemap",
     "clean": "rimraf dist",
     "sitemap": "node bin/build_sitemap.js",
-    "lint": "eslint src --max-warnings 0",
+    "lint": "eslint src --max-warnings 10",
```

Comment:
Code quality: I wasn't the one changing the eslint, just worked on @felixgondwe branch. Having said that, in terms of 'code quality', I think we need to discuss this as some checks are just reflecting opinions and nothing really needed (e.g. never use double quote but only single quote, check the spaces between a } and ) etc). I am fine with them triggering warnings but certainly not errors.

Data flow: this is not using redux and we should be using hooks now that they are there. For the moment, those components store the relevant props into state so that anytime they change, they will tell their children components. There are no real interactions between separated components hence no real use for redux here anyhow. In addition, I was supposed to deliver only the ribbon strips and had to work on the table too, delaying other projects and doing far too much work this week, so no, the data flow won't be changed at this month or at least I won't be the one doing it.